src/features/admin/evaluation-administrations/[id]/evaluees/[evaluation_result_id]/evaluators/evaluators-list.tsx

```diff
@@ -55,6 +55,9 @@ export const EvaluatorsList = () => {
   const [selectedEvaluationId, setSelectedEvaluationId] = useState<number>()
   const isMediumSize = useMobileView(1200)
   const isSmallDevice = useMobileView(800)
+  const [isClick, setIsClick] = useState<boolean>(false)
```

Comment:
Just a small nit, can we rename this to `[checked, setChecked]`? I think it would be clearer and denotes the action of checking

Reply:
done :3